pollutants_analysis1.R

```diff
@@ -0,0 +1,108 @@
+set.seed(123) ## for reproducibility
+library(glmnet)
+library(corrplot)
+data_dir <-  r"(C:\Users\Frank Shi\Documents\FrankS\Waterloo\pollutants)"
+setwd(data_dir)
+pollutants_original <- read.csv(file='pollutants.csv')
+
+#===============================================
+# plot correlation bewtween all covariates
+excl_cols <- c('X', 'length')
+pollutants1 <- pollutants_original[!colnames(pollutants_original) %in% excl_cols]
+pollutants_corr <- cor(pollutants1)
+corrplot(pollutants_corr, order="hclust", method="color", type="full", 
+         tl.col = "black")
+
+# take out a few covariates based on the correlation heat map
+excl_cols1 <- c('X', paste0('POP_PCB', c(1, 2, 4, 5, 7, 9)),
```

Comment:
What is this `X` for?

Reply:
X is the column indexing row numbers, included in the original dataset